rbx_binary/src/serializer.rs

```diff
@@ -696,6 +704,33 @@ impl<'a, W: Write> BinarySerializer<'a, W> {
 
                         chunk.write_interleaved_u32_array(&buf)?;
                     }
+                    Type::Color3uint8 => {
+                        let mut r = Vec::with_capacity(values.len());
+                        let mut g = Vec::with_capacity(values.len());
+                        let mut b = Vec::with_capacity(values.len());
+
+                        for (i, rbx_value) in values {
+                            if let Variant::Color3uint8(value) = rbx_value.as_ref() {
+                                r.push(value.r);
+                                g.push(value.g);
+                                b.push(value.b);
+                            } else {
+                                return type_mismatch(i, &rbx_value, "Color3uint8");
+                            }
+                        }
+
+                        for component in r {
+                            chunk.write_u8(component)?;
+                        }
+
+                        for component in g {
+                            chunk.write_u8(component)?;
+                        }
+
+                        for component in b {
+                            chunk.write_u8(component)?;
+                        }
```

Comment:
Similar to the write examples, we have a method for this from the Rust standard library, [`Write::write_all`](https://doc.rust-lang.org/stable/std/io/trait.Write.html#method.write_all).

We should write something like...

```rust
chunk.write_all(r.as_slice())?;
```

...for each color.